src/main/scala/org/constellation/lb/Manager.scala

```diff
@@ -231,3 +242,18 @@ class Manager(init: NonEmptyList[Addr], config: LoadbalancerConfig)(
       )
   }
 }
+
+object Manager {
+
+  def selectSameSessionNodes(knownSession: Option[Long], currentHosts: NonEmptyMap[Addr, Option[List[Info]]]): (SortedMap[Addr, Option[List[Info]]], SortedMap[Addr, Option[List[Info]]])  = {
+    val (sameSessionNodes, otherSessionNodes) = currentHosts.toSortedMap.partition {
```

Comment:
this is exactly what you return as far as I see, so you don't need to create an additional `val`, do you? 🤔

Reply:
not really, but I thought it was clearer to document which collection is which, but maybe it makes more sense to move to the method's scala doc.

Reply:
ok then :+1: works for me